### ISSUE TITLE
[feat] Expose BE469ZP Parameter 15 (Auto-Lock) as a switch or sensor.

### ISSUE THREAD

**Issue by ledhed-jgh** [zwave-js/zwavejs2mqtt#2181]
**Is your feature request related to a problem? Please describe.**
Prior to using ZWaveJS I could set my Schlage BE469ZP locks "Auto-Lock" feature to on/off via Home Assistant.

Use Cases:
I had automations that would disable Auto-Lock during the day for convenience, like not having to unlock the door when going from my garage to the inside of my home. Then at night I would enable Auto-Lock on all locks for security reasons. I also use it adhoc when there is frequent traffic through a door like when unloading groceries, or when house keepers frequently come in and out the front door ("Hey Google, Housekeeping Mode"). Other use cases would be displaying status on a security dashboard (show which locks don't have Auto-Lock enabled), or turning a smart light red at bed time if a auto-lock was disabled.

**Describe the solution you'd like**
Expose BE469ZP Parameter 15 (Auto-Lock) as a switch or sensor to Home Assistant. A switch would be ideal. A binary sensor would be sufficient to display the status, and you could call the set_config_parameter service to toggle it (more work but doable).

**Describe alternatives you've considered**
I've considered enabling MQTT and creating a MQTT sensor for Parameter 15. The drawback to that is the requirement of a MQTT broker. It also seems excessive to spam the broker with 50+ ZWaveJS devices when all I need is a single parameter from a few locks.

**Additional context**
Parameter 15 is exposed via "Configure Device", but there doesn't appear to be an easy way to monitor its status.

I'd also like to say thank you to everyone who contributes to this project.  ZWaveJS has really made using ZWave practical within Home Assistant.

**Comment by robertsLando** [comment]
@ledhed-jgh Are you using zwavejs integration? BTW the solution you proposed is the only one available at the moment and I don't plan to keep mqtt discovery supported in the future

**Comment by ledhed-jgh** [comment]
Thanks your your quick response.

Yes I use the zwavejs integration, and now that you mention that I realized I probably requested this feature in the wrong place. Sorry about that.

I've turned on MQTT and added the MQTT Sensor manually to my sensors.yaml.  It works, but also have to turn off MQTT Auto Discovery so I don't end up with duplicate entities. I can live with this workaround, but it would still be nice if Auto-Discover was a switch or sensor.  I'm sure there are higher priorities so won't be holding my breath on this one.  :)

Thanks
